### PR TITLE
Set up AWS test_full profile

### PR DESCRIPTION
**Pull request by ewels**
Setting up aws megatests as discussed in https://github.com/nf-core/methylseq/issues/174

@FelixKrueger - do you know how these files should be processed? eg. what kit / trimming / pipeline parameters should be used?

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] `CHANGELOG.md` is updated

### CODE REVIEW

**Comment by FelixKrueger** [comment]
This is bog-standard BS-seq. So Trim Galore and Bismark defaults (with `--ignore_r2 2` I guess).

**Comment by FelixKrueger** [comment]
accidentally clicked on Read for Review, but I don't think there is any harm done

File: conf/test_full.config [hunk 1]
```diff
@@ -12,13 +12,74 @@ params {
...
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961163_GSM3415727_MShef4_J2_BSseq_Homo_sapiens_Bisulfite-Seq_1.fastq.gz',
+        's3://nf-core-awsmegatests/methylseq/input_data/SRR7961163_GSM3415727_MShef4_J2_BSseq_Homo_sapiens_Bisulfite-Seq_2.fastq.gz'
+      ]
+    ]
   ]
-  // Genome references
-  fasta = 'https://github.com/nf-core/test-datasets/raw/methylseq/reference/genome.fa'
-  fasta_index = 'https://github.com/nf-core/test-datasets/raw/methylseq/reference/genome.fa.fai'
+
+  genome = 'GRCh37'
```
Comment (FelixKrueger) [inline conf/test_full.config:66]: Is there still no GRCh38?

**Review by FelixKrueger — COMMENTED** [review]
But apart from the GRCh38 it looks good so far.

File: conf/test_full.config (hunk 1 above)
Comment (ewels) [reply conf/test_full.config:66]: Yeah there is: https://github.com/nf-core/methylseq/blob/4f31ed1792e3cc8ce0ea648b6ee8f541f70102f7/conf/igenomes.config#L26-L37

I think I copied & pasted this from somewhere..

Comment (ewels) [reply conf/test_full.config:66]:
```suggestion
  genome = 'GRCh38'
```

**Comment by ewels** [comment]
ok ace, will merge and run on Tower then we can discuss whether it worked or not on issue #174

**Pull request merged**